src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -83,6 +81,16 @@ public class ServerWideDebugInfoPackageHandler : AdminRequestHandler
                         var databaseNames = ServerStore.Cluster.GetDatabaseNames(transactionOperationContext).ToList();
                         var topology = ServerStore.GetClusterTopology(transactionOperationContext);
 
+                        var operationTimeout = ServerStore.Configuration.Cluster.ClusterOperationTimeout.AsTimeSpan;
+                        var tokenTTL = TimeSpan.FromMilliseconds(operationTimeout.TotalMilliseconds * topology.AllNodes.Count);
+                        var apiKeyToken = SignedTokenGenerator.GenerateToken(
```

Comment:
Why? You are generating a byte array so later you could turn it into string. Turn it into string directly.